src/components/manager/projects/ProjectJsonSchema.js

```diff
@@ -76,13 +76,7 @@ const ProjectJsonSchema = ({ projectSchema }) => {
                 </Descriptions.Item>
 
                 <Descriptions.Item label="JSON Schema">
-                    <ReactJson
-                        src={projectSchema.json_schema ?? {}}
-                        collapsed={true}
-                        displayDataTypes={false}
-                        enableClipboard={false}
-                        name={null}
-                    />
+                    <JsonView src={projectSchema.json_schema ?? {}} collapsed={true} />
```

Comment:
`JsonView`'s `collapsed` prop is true by default.

Reply:
it's 1 by default, which means 1 level of nesting is expanded (the top level)